docs/java-application-development/kura-addon-archetype.md

```diff
@@ -135,4 +135,348 @@ Finally, configure the external **Classpath** dependencies by selecting the plug
 
 ![](./images/kura-addon-archetype/eclipse-add-library.png)
 
-On old Eclipse IDE installations it might be necessary to uninstall the Tycho configurator 0.1.0 plugin from _Help | Install new software... | What is already installed?_.
\ No newline at end of file
+On old Eclipse IDE installations it might be necessary to uninstall the Tycho configurator 0.1.0 plugin from _Help | Install new software... | What is already installed?_.
+
+## Architecture-specific development
+
+The Addon Archetype standard procedure allows to build generic debian and RPM installers not dependant on the architecture on which the bundle will be installed.
+
+However, it is possible to adjust the files in the `distrib` folder to develop architecture-dependant installers: it could be necessary to build a bundle that contains native code (C/C++ libraries, jars that use JNI, etc.) or to build a bundle that contains architecture-specific files (e.g. a systemd service file). In this case, the `distrib` folder should be modified to include the architecture-specific files in the generated package.
```

Comment:
```suggestion
However, it is possible to customise the files in the `distrib` folder to develop architecture-dependant installers: this might be necessary when a bundle contains native code (C/C++ libraries, jars that use JNI, etc.) or architecture-specific files (e.g. a systemd service file).
```